src/hor2eq.jl

```diff
@@ -0,0 +1,137 @@
+# This file is a part of AstroLib.jl. License is MIT "Expat".
+
+function _hor2eq(alt::T, az::T, jd::T, ws::Bool, B1950::Bool,
+                 precession::Bool, nutate::Bool, aberration::Bool,
+                 refract::Bool, lat::T, lon::T, altitude::T, pressure::T,
+                 temperature::T, obsname::AbstractString) where {T<:AbstractFloat}
+
+    if obsname == ""
+        # Using Pine Bluff Observatory values
+        if isnan(lat)
+            lat = T(43.0783)
+        end
+
+        if isnan(lon)
+            lon = T(-89.865)
+        end
+    else
+        lat = observatories[obsname].latitude
+        lon = observatories[obsname].longitude
+        altitude = observatories[obsname].altitude
+    end
+
+    if refract
+        alt_b = co_refract(alt, altitude, pressure, temperature)
+    else
+        alt_b = alt
+    end
+
+    if ws
+        az -= 180
+    end
+    dra1, ddec1, eps, d_psi, _ = co_nutate(jd, 45, 45)
+    last = 15 * ct2lst(lon, jd) + d_psi * cos(eps) / 3600
+    ha, dec = altaz2hadec(alt_b, az, lat)
+    ra = mod(last - ha, 360)
+    ha /= 15
```

Comment:
I don't think `ha` is returned in degrees, not hours, this instruction should be removed (see comment about the result of the test below)